### DISCUSSION TITLE
Sorry Delete this. Add the F1TV stream commentary in addition to English commentary

### DISCUSSION THREAD

**Discussion by woutercommandeur** [f1multiviewer/issue-tracker#287] [Feature requests] [answered]
F1 also has international english audio. Currently English gives the Sky Sports English commentary, I much prefer the F1TV English commentary.

Sorry, ignore this: I should have clicked the F1 live button instead of International... close this.

**Comment by f1multiviewer** [comment, marked as answer]
Hey, you can get this already, just open the "F1 Live" instead of "International", and you'll get the F1TV commentary team
